base/bsupport/logcustomcounterstub.go

```diff
@@ -1,17 +1,54 @@
 package bsupport
 
 import (
+	"fmt"
+
 	"github.com/relex/slog-agent/base"
 )
 
 type stubLogCustomCounterRegistry struct {
+	lookup map[string]*stubLogCustomCounter
+}
+
+type stubLogCustomCounter struct {
+	count  int64
+	length int64
 }
 
+// LookupStubCustomerCounterFunc can be called to get (log count, total log length) by registered label name
+//
+// For testing only
```

Comment:
Could we put utilities that are only used for testing in a separate subpackage, like for example the [http library does](https://pkg.go.dev/net/http/httptest)?